RecoHI/HiEvtPlaneAlgos/BuildFile.xml

```diff
@@ -1,23 +1,17 @@
 <use   name="root"/>
 
 <use   name="FWCore/Framework"/>
-<use   name="FWCore/PluginManager"/>
 <use   name="FWCore/ParameterSet"/>
 <use   name="FWCore/Utilities"/>
-<use   name="PhysicsTools/UtilAlgos"/>
 <use   name="DataFormats/Candidate"/>
 <use   name="DataFormats/HcalRecHit"/>
-<use   name="DataFormats/WrappedStdDictionaries"/>
-<use   name="Geometry/Records"/>
-<use   name="DataFormats/HepMCCandidate"/>
 <use   name="DataFormats/TrackReco"/>
 <use   name="DataFormats/VertexReco"/>
 <use   name="DataFormats/Common"/>
 <use   name="DataFormats/HeavyIonEvent"/>
 <use   name="DataFormats/CaloTowers"/>
-<use   name="DataFormats/Provenance"/>
 <use   name="CondCore/DBOutputService"/>
 <use   name="CondFormats/DataRecord"/>
-<use   name="RecoHI/HiCentralityAlgos"/>
-<use   name="clhep"/>
+<use   name="SimDataFormats/TrackingAnalysis"/>
```

Comment:
this looks like a fake dependency.
May I ask you to cleanup the includes in RecoHI/HiEvtPlaneAlgos/src/HiEvtPlaneFlatProducer.cc which apparently does not need anything from Sim\*.

Reply:
You are right, these Sim dependencies in Reco are not really good. I'll remove it.